lib/utils/packaging/unarchive.go

```diff
@@ -131,7 +132,9 @@ func replaceZip(toolsDir string, archivePath string, extractDir string, execName
 			if err := os.Remove(appPath); err != nil && !os.IsNotExist(err) {
 				return trace.Wrap(err)
 			}
-			if err := os.Symlink(dest, appPath); err != nil {
+			// For the Windows build we have to use hard links to be able
+			// to use client tools without administrative access.
+			if err := os.Link(dest, appPath); err != nil {
```

Comment:
Can we avoid hardlinks outside of Windows? They don't work across mount points, they don't work on all file systems, and they can lead to bugs that are hard to predict.

If they are necessary Windows, we should make sure that `extractDir` and `toolsDir` are on the same volume before we start extracting. Note that FAT/exFAT don't support them, so we may need a fallback behavior.

Reply:
I would probably just replace this one with copy for the windows platform only, otherwise with symlinks we always require administrative access